plugins/woocommerce/client/legacy/css/admin.scss

```diff
@@ -7529,40 +8097,55 @@ table.bar_chart {
 	}
 }
 
+@media only screen and (max-width: 1280px) {
+	#woocommerce-product-data,
+	#woocommerce-coupon-data {
+		.options_group,
+		// General coupos data does not have options group wrapper, make it look like it has.
```

Comment:
Typo: coupos

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/52632/commits/153bf863da0e1e18882872f97cc7744ca964bcc5